src/ol/renderer/canvas/tilelayer.js

```diff
@@ -1,13 +1,10 @@
-// FIXME find correct globalCompositeOperation
```

Comment:
Is the above fixed by this PR, or just no longer relevant?

Reply:
We're using the correct `globalCompositeOperation` already (`'source-over'`).  There are situations where `'copy'` is useful, but not in this code where we are only rendering to a portion of the canvas at a time.